index.rst

```diff
@@ -75,41 +78,73 @@ A general overview of the development flow, as viewed from the user, is as follo
 
     - Request new functionality in lower-level control classes (e.g. attcs)
 
-    - Migrate/evolve the utilities and Job to a production level for regular use.
+    - Migrate/evolve the utilities and Job to a production level for regular use with the Queue
+
+How the workflow moves through the various repositories is represented by the following diagram, where each of the
+sections is discussed in detail below.
+
+.. figure:: _static/notebook_and_queueJob_workflow_diagram.jpg
+    :width: 600px
+    :align: center
+    :alt: Workflow Summary
+
+    A visual overview of the workflow process, starting from an original idea first demonstrated in a Jupyter notebook.
+
+The two-tiered development strategy is being adopted to facilitate maximum flexibility to implement changes on
+short timescales. The staging (or development) area gives users full range to simultaneously operate across multiple
```

Comment:
The development and staging areas gives users full range to simultaneously operate across multiple